docs/Icons.md

```diff
@@ -49,6 +55,21 @@ import { Battery } from '@pxblue/react-native-progress-icons';
 />
 ```
 
+### API
+
+<div style="overflow: auto">
+
+| Prop Name        | Description                                                                     | Type               | Required | Default |
+| ---------------- | ------------------------------------------------------------------------------- | ------------------ | -------- | ------- |
+| size             | What size the icon / component should be                                        | `number`           | no       |         |
+| color            | What color the icon / component should be                                       | `string`           | no       |         |
+| allowFontScaling | Should the icon / component size scale with the system font size                | `boolean`          | no       |         |
+| direction        | The current device language direction (for flipping components where necessary) | `'ltr'` \| `'rtl'` | no       |         |
+
+</div>
+
+PX Blue components will pass these values to you for use in your component to achieve the correct appearance. It is your responsibility to apply them.
+
 ## Wrapped Icon
```

Comment:
```suggestion
## Wrapped Icon (Deprecated)
```

Reply:
It's not _actually_ deprecated at this time, but it might be in the future.